docs/design/node-selection.md

```diff
@@ -48,12 +48,12 @@ This design may be refined in the future to prefer storage nodes based on speed,
 ```DBX
 model node (
 ...
-	field audit_reputation_α   float64   ( updatable )
-	field audit_reputation_β   float64   ( updatable )
-	field total_audit_count    int64     ( updatable )
-	field uptime_reputation_α  float64   ( updatable )
-	field uptime_reputation_β  float64   ( updatable )
-	field total_uptime_count   int64     ( updatable )
+	field reputation_audit_alpha  float64   ( updatable )
```

Comment:
this is very nitpicky and okay as is, but I prefer `audit_reputation_alpha` and `uptime_reputation_alpha` because I find it easier to distinguish the fields when they're all in a list.
e.g.
```
			dbx.Node_AuditReputationAlpha(1),
			dbx.Node_AuditReputationBeta(0),
			dbx.Node_UptimeReputationAlpha(1),
			dbx.Node_UptimeReputationBeta(0),
```
vs
```
			dbx.Node_ReputationAuditAlpha(1),
			dbx.Node_ReputationAuditBeta(0),
			dbx.Node_ReputationUptimeAlpha(1),
			dbx.Node_ReputationUptimeBeta(0),
```
idk hopefully other people also perceive a difference 😂

Reply:
actually, a better supporting argument for this suggestion is the fact that the NodeStats fields are `AuditReputationAlpha`, `UptimeReputationAlpha`, etc. in this PR (https://github.com/storj/storj/pull/2226), so I think it seems reasonable for them to be consistent.

Reply:
sure!  and fixed!